src/main/java/com/redhat/devtools/lsp4ij/launching/templates/LanguageServerTemplateManager.java

```diff
@@ -40,13 +40,9 @@ public static LanguageServerTemplateManager getInstance() {
         return ApplicationManager.getApplication().getService(LanguageServerTemplateManager.class);
     }
 
+    // TODO: This should not be necessary anymore. Loading serves from folders

```

Comment:
Not quite sure how this should be handled now with TelemetryManager as there shouldn't be any static templates any more.

Reply:
@fbricon what do you think about add telemtry for export / import of language servers?